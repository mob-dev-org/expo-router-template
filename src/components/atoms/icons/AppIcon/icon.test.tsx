// import React from 'react';
// import { render } from '@testing-library/react-native';

// import Icon from './index';

// describe('Icon', () => {
//     it('When passing one of the names to return Icon and match snapshot', () => {
//         const component = render(<Icon name="back" />);
//         expect(component).toMatchSnapshot();
//     });

//     it('When passing stroke use it in icon', () => {
//         const component = render(<Icon name="bell" color={'#ccc'} />);
//         expect(component).toMatchSnapshot();
//     });

//     it('When adding style use it in icon', () => {
//         const component = render(<Icon name="bell" style={{ marginTop: 16 }} />);
//         expect(component).toMatchSnapshot();
//     });
// });
