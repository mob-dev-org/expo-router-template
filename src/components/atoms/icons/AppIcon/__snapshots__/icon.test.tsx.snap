// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon When adding style use it in icon 1`] = `
<SvgrURL
  color="black"
  height={24}
  style={
    {
      "marginTop": 16,
    }
  }
  testID="bellIcon"
  width={24}
/>
`;

exports[`Icon When passing one of the names to return Icon and match snapshot 1`] = `
<SvgrURL
  color="black"
  height={24}
  testID="backIcon"
  width={24}
/>
`;

exports[`Icon When passing stroke use it in icon 1`] = `
<SvgrURL
  color="#ccc"
  height={24}
  testID="bellIcon"
  width={24}
/>
`;
